packages/x-charts-premium/src/index.ts

```diff
@@ -0,0 +1,51 @@
+// TODO: Add typeOverloads when available
+// import './typeOverloads/modules';
+
+// exports from MIT package
+export * from '@mui/x-charts/constants';
+export * from '@mui/x-charts/context';
+export * from '@mui/x-charts/hooks';
+export * from '@mui/x-charts/colorPalettes';
+export * from '@mui/x-charts/models';
+export * from '@mui/x-charts/ChartsClipPath';
+export * from '@mui/x-charts/ChartsReferenceLine';
+export * from '@mui/x-charts/ChartsAxis';
+export * from '@mui/x-charts/ChartsXAxis';
+export * from '@mui/x-charts/ChartsYAxis';
+export * from '@mui/x-charts/ChartsGrid';
+export * from '@mui/x-charts/ChartsText';
+export * from '@mui/x-charts/ChartsTooltip';
+export * from '@mui/x-charts/ChartsLegend';
+export * from '@mui/x-charts/ChartsLocalizationProvider';
+export * from '@mui/x-charts/ChartsAxisHighlight';
+export * from '@mui/x-charts/BarChart';
+export * from '@mui/x-charts/LineChart';
+export * from '@mui/x-charts/PieChart';
+export * from '@mui/x-charts/ScatterChart';
+export * from '@mui/x-charts/SparkLineChart';
+export * from '@mui/x-charts/Gauge';
+export * from '@mui/x-charts/RadarChart';
+export * from '@mui/x-charts/ChartsSurface';
+export * from '@mui/x-charts/ChartDataProvider';
+export * from '@mui/x-charts/ChartsLabel';
+export type {
+  ZoomData,
+  ZoomFilterMode,
+  ZoomSliderShowTooltip,
+  ZoomOptions,
+  ZoomSliderOptions,
+} from '@mui/x-charts/internals';
+
+// Pro components
+export * from '@mui/x-charts-pro/hooks';
+export * from '@mui/x-charts-pro/Heatmap';
+export * from '@mui/x-charts-pro/ChartContainerPro';
+export * from '@mui/x-charts-pro/ChartDataProviderPro';
+export * from '@mui/x-charts-pro/ScatterChartPro';
+export * from '@mui/x-charts-pro/BarChartPro';
+export * from '@mui/x-charts-pro/LineChartPro';
+export * from '@mui/x-charts-pro/PieChartPro';
+export * from '@mui/x-charts-pro/FunnelChart';
+export * from '@mui/x-charts-pro/RadarChartPro';
+export * from '@mui/x-charts-pro/ChartZoomSlider';
+export * from '@mui/x-charts-pro/ChartsToolbarPro';
```

Comment:
I''m wondering if this would be the same
```suggestion
export * from '@mui/x-charts-pro'
```

Reply:
Eventually we would have to do the extensive one to remove things like `BarChartPro` if we have `BarChartPremium` no?

Reply:
I'm not sure we will because we already export both BarChart and BarChartPro from the pro package.

If I remember correctly the argume was to let user still use the community bar chart for their small usecases